jujugui/static/gui/src/app/components/inspector/config/config.js

```diff
@@ -140,6 +140,14 @@ class Configuration extends React.Component {
         });
         return;
       }
+      if (!nameValue) {
```

Comment:
While this solves the immediate problem, why don't we expand on this a bit and test with the same regex as core so we're also sure it's a valid name wrt the allowed characters?